packages/client/src/styles/globals.css

```
@@ -888,6 +888,11 @@
   stroke: url(#mari-topbar-chats-gradient);
 }
 
+.mari-topbar .mari-topbar-accent-icon {
+  color: var(--marinara-chat-chrome-button-text-active);
+  stroke: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)